app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -66,10 +67,16 @@ protected function validator(array $data)
      */
     protected function create(array $data)
     {
+        $settings = null;
+
+        if (Session::get('invite_token')) {
```

Comment:
We have other code that fills out the "settings" table when a user registers after clicking on a google ad. Please move the logic to the same place to keep the code clean :)